### DISCUSSION TITLE
software for pixel de-identification of images

### DISCUSSION THREAD

**Discussion by gkowalski** [RSNA/anonymizer#9] [General]
What are other people using for De-identification of image data ? I've been using the https://www.xnat.org/download/upload-assistant/ for years and have had increasing reliability issues with it and am looking for other options.

**Comment by mdevans** [comment]
Dear George,

The next update to the RSNA/anonymizer will include a pixel anonymizer which will attempt to remove all typed and
handwritten text on images.

I will notify the CTP group when it will be available. The workstation will need decent compute resources and ±16GB RAM for the pixel anonymizer algorithms to work efficiently.

Michael Evans

On Sat, 21 Sept 2024 at 17:42, George Kowalski ***@***.***>
wrote:

> What are other people using for De-identification of image data ? I've
> been using the https://www.xnat.org/download/upload-assistant/ for years
> and have had increasing reliability issues with it and am looking for other
> options.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RSNA/anonymizer/discussions/9>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIXFIFFD5AN23BV3VJVRYDZXWHXFAVCNFSM6AAAAABOTWCNA6VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGIYTIOBXGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Michael Evans
*DX.LIFE (Pty) Ltd.*
+27 72 584 6989

**Reply by gkowalski** [reply]
I have 96 Gig of Memory ! Bring it on . Happy to help out test any release.

**Comment by mdevans** [comment]
Excellent :) Thanks, that would be really great, will send you the initial
release. Please email me at ***@***.***

On Mon, 23 Sept 2024 at 21:13, George Kowalski ***@***.***>
wrote:

> I have 96 Gig of Memory ! Bring it on . Happy to help out test any release.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/RSNA/anonymizer/discussions/9#discussioncomment-10730382>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABIXFIHNVVX3NGE47VZ653LZYBR5LAVCNFSM6AAAAABOTWCNA6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANZTGAZTQMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Michael Evans
*DX.LIFE (Pty) Ltd.*
+27 72 584 6989
